guides/common/modules/proc_customizing-the-discovery-pxe-boot.adoc

```diff
@@ -0,0 +1,44 @@
+[id="Customizing_the_Discovery_PXE_Boot_{context}"]
+= Customizing the Discovery PXE boot
+[[Discovery_Templates_and_Snippets_Settings_{context}]]
+[[Provisioning_Template_PXELinux_Discovery_Snippet_{context}]]
+
+{Project} builds PXE boot menus from the following global provisioning templates:
+
+* `PXELinux global default` for BIOS provisioning.
+* `PXEGrub global default` and `PXEGrub2 global default` for UEFI provisioning.
+
+The PXE boot menus are available on {ProjectServer} and {SmartProxies} that have TFTP enabled.
+
+The Discovery menu item uses a Linux kernel for the operating system and passes kernel parameters to configure the Discovery service.
+You can customize the passed kernel parameters by changing the following snippets:
+
+* `pxelinux_discovery`: This snippet is included in the `PXELinux global default` template.
++
+This snippet renders the Discovery boot menu option.
+The `KERNEL` and `APPEND` options boot the Discovery kernel and initial RAM disk.
+The `APPEND` option contains kernel parameters.
+
+* `pxegrub_discovery`: This snippet is included in the `PXEGrub global default` template.
+However, Discovery is http://projects.theforeman.org/issues/15997[not implemented for GRUB 1.x].
```

Comment:
```suggestion
ifndef::orcharhino[]
However, Discovery is http://projects.theforeman.org/issues/15997[not implemented for GRUB 1.x].
endif::[]
```

Reply:
Grub2 is over ten years old: GNU GRUB version 2.00 was officially released on June 26, 2012.[[17]](https://en.wikipedia.org/wiki/GNU_GRUB#cite_note-19)[[18]](https://en.wikipedia.org/wiki/GNU_GRUB#cite_note-20) maybe we can drop this altogether? Nobody worked on this for eight years according to the issue, so it's probably not too important?

see https://en.wikipedia.org/wiki/GNU_GRUB#Development

Reply:
The real question is how likely the user is to need to use Grub 1 instead of Grub 2. I'm not sure in what circumstances they would need it. I would even consider not documenting any templates/snippets for Grub 1 if it's unlikely.

Reply:
I don't know, but to me, this sounds unlikely that someone still uses grub1.

Reply:
asked internally: :+1: to dropping the note for grub1.